tests/testthat/test-build-home-community.R

```diff
@@ -0,0 +1,25 @@
+context("test-build-home-community.R")
+
+test_that("community section is added if COC present", {
```

Comment:
This test construction doesn't feel right to me, because you're not directly calling any functions in the `build-home-community.R` file

Reply:
Right, I re-wrote the tests. :-)

Reply:
Could you also add direct tests for `has_coc()` and `has_contributing()`

Reply:
done, but there's no site with a contributing file in the assets, correct?

Reply:
That's correct, testing the one is fine.

Reply:
great, done.